calyx-opt/src/default_passes.rs

```diff
@@ -36,10 +36,8 @@ impl PassManager {
         pm.register_pass::<GroupToSeq>()?;
         pm.register_pass::<InferShare>()?;
         pm.register_pass::<CellShare>()?;
-        pm.register_pass::<InferStaticTiming>()?;
-        pm.register_pass::<MergeStaticPar>()?;
+        pm.register_pass::<StaticPromotion>()?;
         pm.register_pass::<SimplifyStaticGuards>()?;
-        pm.register_pass::<StaticParConv>()?;
         pm.register_pass::<DataPathInfer>()?;
```

Comment:
Woo! Fewer passes! Remember to delete the files associated with these passes too